packages/docs/docs/concepts/cornerstone-tools/touch.md

```diff
@@ -0,0 +1,156 @@
+| `TOUCH_TAP`            | Triggers when the user makes contact with screen for less than 50ms - 10ms (buffer from the `TOUCH_START` event.)                                                                                |
+
+## Multitouch
+
+Touch events natively support multitouch which is provided as a list of [`ITouchPoints[]`](api/tools/namespace/Types#ITouchPoints).
+In order for touch events to be compatiable with mouse events, these `ITouchPoints[]` need to be reduced into a single
+`ITouchPoint`. The current strategy for array reduction is taking the mean coordinate values. Other strategies can be
+implemented such as first point, median point, etc. This can be implemented in the
+[`touch` utilities codebase](https://github.com/cornerstonejs/cornerstone3D-beta/blob/gradienthealth/main/packages/tools/src/utilities/touch/index.ts)
+
+The structure of `ITouchPoints` are the following:
+
+```js
+type ITouchPoints = {
+  /** page coordinates of the point */
+  page: Types.Point2,
+  /** client coordinates of the point */
+  client: Types.Point2,
+  /** canvas coordinates of the point */
+  canvas: Types.Point2,
+  /** world coordinates of the point */
+  world: Types.Point3,
+
+  /** Native Touch object properties which are JSON serializable*/
+  touch: {
+    identifier: string,
+    radiusX: number,
+    radiusY: number,
+    force: number,
+    rotationAngle: number,
+  },
+};
+```
+
+## Multitouch Drag Calculations
+
+`TOUCH_DRAG` events have the following structure:
+
+```js
+type TouchDragEventDetail = NormalizedTouchEventDetail & {
+  /** The starting points of the touch event. */
+  startPoints: ITouchPoints,
+  /** The last points of the touch. */
+  lastPoints: ITouchPoints,
+  /** The current touch position. */
+  currentPoints: ITouchPoints,
+  startPointsList: ITouchPoints[],
+  /** The last points of the touch. */
+  lastPointsList: ITouchPoints[],
+  /** The current touch position. */
+  currentPointsList: ITouchPoints[],
+
+  /** The difference between the current and last points. */
+  deltaPoints: IPoints,
+  /** The difference between distances between the current and last points. */
+  deltaDistance: IDistance,
+};
+```
+
+`deltaPoints` is the difference between the mean coordinate point of `lastPointsList` and `currentPointsList`.
+`deltaDistance` is the difference between the average distance between points in `lastPointsList` vs `currentPointsList`
+
+## Usage
+
+You can add an event listener to the element for the event.
+
+```js
+import Events from '@cornerstonejs/tools/enums/Events';
+// element is the cornerstone viewport element
+element.addEventListener(Events.TOUCH_DRAG, (evt) => {
+  // my function on drag
+  console.log(evt);
+});
+
+element.addEventListener('CORNERSTONE_TOOLS_SWIPE', (evt) => {
```

Comment:
it would be great if these events are also `EVENTS.`

Reply:
I think these are included as `EVENTS.` but for doc purposes I just use the string for simplicity. Can def change if needed.

Reply:
Yes please, I highly prefer consistency in all places

Reply:
Update to `CORNERSTONE_TOOLS_SWIPE`  => `Events.[CORNERSTONE_TOOLS_SWIPE]`